db/schema.rb

```diff
@@ -10,14 +10,17 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_01_05_030246) do
+ActiveRecord::Schema.define(version: 2019_01_14_043024) do
 
   create_table "users", force: :cascade do |t|
     t.string "nickname"
     t.string "image"
     t.text "introduction"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.string "image_name"
+    t.string "name"
+    t.string "image_age"
```

Comment:
@constantino7kotaro 
こちらimage_ageのままになってますので、再度migrateファイルを作成してimage_ageをageに変える必要があります。

Reply:
修正後にpushしようとしたら、以下のようなエラーメッセージが出ました

To https://github.com/constantino7kotaro/tobira.git
 ! [rejected]        user_profile_edit -> user_profile_edit (non-fast-forward)
error: failed to push some refs to 'https://github.com/constantino7kotaro/tobira.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

Reply:
@constantino7kotaro 
こちらエラーログを読んでググってみたでしょうか？
例えば、
` Updates were rejected because the tip of your current branch is behind `
や
` Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again. `
と記載があります。

実際にググるとコンフリクトを起こしているためエラーになっていることがわかると思います。
ブランチを最新化してコンフリクトの解消をしてからpushをしてみてください。
画像はブランチをマージするボタンの近くにあるgit hubの画面です。

<img width="695" alt="2019-01-23 0 20 44" src="https://user-images.githubusercontent.com/29660904/51545199-d051c380-1ea4-11e9-854e-ffbc77c1b03f.png">

Reply:
色々試しましたが正解分かりません！
なにをどうすればいのか一から教えてください。
なかなか先に進めずにいます。

Reply:
マスターを最新化するというのは、 ブランチをmasterに切り替えてgit pull origin master でいいのですか？
そのあとにブランチをuser_profile_editに切り替えて git merge master とコマンド打ってから、add, commit, pushをするのでしょうか？

Reply:
>マスターを最新化するというのは、 ブランチをmasterに切り替えてgit pull origin master でいいのですか？
そのあとにブランチをuser_profile_editに切り替えて git merge master とコマンド打ってから、add, commit, pushをするのでしょうか？

はいそうです。前に添付した画像が示すようにコンフリクトが必ず起きるので修正してから add commit pushをお願いします。
@constantino7kotaro

Reply:
試しましたができません。。。
=> 前に添付した画像が示すようにコンフリクトが必ず起きるので修正してから add commit pushをお願いします。
「コンフリクトを修正してから」というのは、画像の[resolve conflict]ボタンを押して、内容が出て来て、それ通りにmasterブランチのuser_controllerを修正するということですか？（そうしました）　そのあとに、masterブランチのadd, commit, pushをして（実際にしました）、ブランチをuser_profile_editに切り替えて、add, commit, pushすればいいんですよね？（しました）
それでも同じエラーメッセージが出て来て、user_profile_editをpushできません。

Reply:
https://qiita.com/te2u/items/c23f82ec84cf65564554
記事参考にして色々試しましたが解決せず。また明日にします

Reply:
本日の授業で再度コンフリクト解消の手順をご説明します。それ前提以下残しておきます。

前回の授業でもコンフリクトの解消を行いました(showをマージする際のことです)が、resolve conflictボタンを使っていないはずです。そのため

>「コンフリクトを修正してから」というのは、画像の[resolve conflict]ボタンを押して
これは間違いです。この機能を使っても確かにコンフリクトは解消できますが上級者もしくはレビュワーが使うものです。あくまであの画像はコンフリクトがおきていることをgithubが知らせてくれているものという認識でお願いします。

>masterブランチのuser_controllerを修正するということですか？
こちら授業で教えましたが、masterブランチでソースコードはいじることはないですし、チーム開発においてはご法度です。授業前にどのような操作をしたのかまとめていただけるとスームズかと思います。

コンフリクトは、坂村さんが見つけた記事の通りmasterとブランチ間で起きるものです。そしてコンフリクトを解消するのはmasterではなくブランチで解消するものです。
前回授業で、showブランチでコンフリクトを解消しました。

>それでも同じエラーメッセージが出て来て、user_profile_editをpushできません。
masterにソースコードをadd commit pushをしてしまったためmasterとuser_profile_edtiで再度コンフリクトが起きているのが現状です。